packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -150,8 +154,7 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
               metadata: {
                 ...metadata,
                 apps: {
-                  ...metadata?.apps,
-                  [appSlug]: null,
```

Comment:
When uninstalling an event type app, by setting the app object to `null` it causes a zod error which means the event type page won't load. Instead we should be removing the app from the event type.